community/kernel/src/main/java/org/neo4j/kernel/impl/util/function/README

```diff
@@ -1 +0,0 @@
-This package is being replaced in favor of org.neo4j.function.
```

Comment:
Is there nothing else left in the package? Otherwise this could be a `package-info.java`